share/spice/yoga_asanas/yoga_asanas.css

```diff
@@ -0,0 +1,15 @@
+.zci--yoga_asanas img {
+	max-width: none;
+}
+
+.zci--yoga_asanas .tile__media__img {
+	height: 125%;
+	position: absolute;
+	bottom: -10px;
+	left: 50%;
```

Comment:
Seems like removing `left:50%` and `transform: ...` cause the display to be the same? Are you sure we need them?

Either way, it's best to just use `left` because the transform might not work everywhere. CanIUseIt suggests it's not supported in OperaMini -- not sure how many users we have using that browser though...